### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Fixes misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

The misspellings have been reported at https://github.com/jsoref/gittuf/actions/runs/6053649280#summary-16429561029

The action reports that the changes in this PR would make it happy:  https://github.com/jsoref/gittuf/actions/runs/6053649491#summary-16429561645

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -10,7 +10,7 @@ updates:
       prefix: "chore"
       include: "scope"
     open-pull-requests-limit: 10
-  # Monitor Github Actions
+  # Monitor GitHub Actions
```
Comment (jsoref) [inline .github/dependabot.yml:13]: brand

File: internal/signerverifier/dsse/dsse.go [hunk 2]
```diff
@@ -56,7 +56,7 @@ func SignEnvelope(ctx context.Context, envelope *dsse.Envelope, signer dsse.Sign
 }
 
 // VerifyEnvelope verifies a DSSE envelope against an expected threshold using
-// a slice of verifiers passed in to it. Threshold indicates the number of
+// a slice of verifiers passed into it. Threshold indicates the number of
```
Comment (jsoref) [inline internal/signerverifier/dsse/dsse.go:59]: `in to` can be correct if the thing after `to` is a verb, but it isn't in this case.

**Review by adityasaky — APPROVED** [review]

**Comment by adityasaky** [comment]
I'm guessing this is a bot but thanks regardless!

**Comment by jsoref** [comment]
I'm human. Really!

**Comment by adityasaky** [comment]
Huh, wow! Well, thank you!

**Pull request merged**